packages/office-ui-fabric-react/src/components/TeachingBubble/examples/TeachingBubble.Condensed.Example.tsx

```diff
@@ -25,16 +23,17 @@ export class TeachingBubbleCondensedExample extends React.Component<{}, ITeachin
 
     return (
       <div className="ms-TeachingBubbleExample">
-        <span className="ms-TeachingBubbleBasicExample-buttonArea" ref={menuButton => (this._menuButtonElement = menuButton!)}>
+        <span className="ms-TeachingBubbleBasicExample-buttonArea">
           <DefaultButton
             onClick={isTeachingBubbleVisible ? this._onDismiss : this._onShow}
             text={isTeachingBubbleVisible ? 'Hide TeachingBubble' : 'Show TeachingBubble'}
+            id={'buttonId'}
           />
         </span>
         {isTeachingBubbleVisible ? (
           <div>
             <TeachingBubble
-              targetElement={this._menuButtonElement}
+              target={'#buttonId'}
```

Comment:
Do we want to do this? Since this is in our examples I think we should not use this approach if we don't want to encourage it.

Reply:
I think that we should have one example that does this since the callout provides for all of those options. I think it's okay to encourage it. @micahgodbolt do you have an opinion?

Reply:
string based target is, AFAIK, prefered over creating an entire reference and using the reference. Its certainly less code for the user. I have no problem showing this example and would wonder if all of our target examles should be this way. One improvement might be to generate a unique ID on page load, rather than a static string.

Reply:
I think we should maybe show the gambit, at least in callout since it does accept a bunch